tests/driver.c

```diff
@@ -16,6 +17,32 @@ struct timespec start_time, end_time;
 char driver_tag_msg[NUM_TRYTES_TAG];
 ta_send_mam_res_t* res;
 
+typedef status_t (*Proxy_apis)(const iota_client_service_t* const service, const char* const obj, char** json_result);
+typedef status_t (*Proxy_apis_without_args)(const iota_client_service_t* const service, char** json_result);
+
+static struct proxy_apis_s {
+  Proxy_apis api;                            // function pointer of IRI Proxy API with args
+  Proxy_apis_without_args api_without_args;  // function pointer of IRI Proxy API without args
+  const char* name;
+  const char* json;  // args which are passed to IRI API
+} proxy_apis_g[] = {{api_check_consistency, NULL, "check_consistency",
```

Comment:
Why did you name it `proxy_apis_g`?

Reply:
The naming rule of `proxy_apis_g` follows the naming rule of `ta_cli_arguments_g` in [accelerator/message.h](https://github.com/DLTcollab/tangle-accelerator/blob/develop/accelerator/message.h#L53).